docs/brainstorms/2026-05-07-audit-followups-brainstorm.md

```
@@ -0,0 +1,242 @@
+### Apply items with acceptance criteria
+
+#### C-02 — Fix legacy 2-segment subagent_types in `plan.md`
+- **File:** `plugins/yellow-core/commands/workflows/plan.md` lines 90, 98, 132
+- **Changes:**
+  - `yellow-core:repo-research-analyst` → `yellow-core:research:repo-research-analyst`
+  - `yellow-core:best-practices-researcher` → `yellow-core:research:best-practices-researcher`
+  - `yellow-core:spec-flow-analyzer` → `yellow-core:workflow:spec-flow-analyzer`
+- **Verify:** `pnpm validate:agents` shows no INFO warnings on these three values
+- **Changeset:** patch bump on yellow-core
+
+#### X-02 — Fix validator CHANGELOG false positive
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)